### PR TITLE
Update documentation in methods

### PR DESCRIPTION
**Pull request by CaedenPH**
Add documentation to certain methods within the `ws` and `http` clients.

Some methods are left blank if I was unsure of what they do.

### CODE REVIEW

File: surrealdb/clients/ws.py [hunk 1]
```diff
@@ -82,14 +100,13 @@ async def connect(self) -> None:
 
         if self._token is not None:
             await self.authenticate(self._token)
-
```
Comment (tsunyoku) [inline surrealdb/clients/ws.py:85]: why are we removing these whitespaces? it separates out the logic and removing them is harder to read

File: surrealdb/clients/ws.py [hunk 2]
```diff
@@ -82,14 +100,13 @@ async def connect(self) -> None:
...
         if self._username is not None and self._password is not None:
             await self.signin(username=self._username, password=self._password)
-
```
Comment (tsunyoku) [inline surrealdb/clients/ws.py:88]: why are we removing these whitespaces? it separates out the logic and removing them is harder to read

File: surrealdb/clients/ws.py [hunk 3]
```diff
@@ -203,13 +237,10 @@ async def signin(
 
         if username is not None:
             self._username = username
-
```
Comment (tsunyoku) [inline surrealdb/clients/ws.py:206]: why are we removing these whitespaces? it separates out the logic and removing them is harder to read

File: surrealdb/clients/ws.py [hunk 4]
```diff
@@ -203,13 +237,10 @@ async def signin(
...
         if password is not None:
             self._password = password
-
```
Comment (tsunyoku) [inline surrealdb/clients/ws.py:209]: why are we removing these whitespaces? it separates out the logic and removing them is harder to read

File: surrealdb/clients/ws.py [hunk 5]
```diff
@@ -203,13 +237,10 @@ async def signin(
...
         if namespace is not None:
             self._namespace = namespace
-
```
Comment (tsunyoku) [inline surrealdb/clients/ws.py:212]: why are we removing these whitespaces? it separates out the logic and removing them is harder to read

File: surrealdb/clients/http.py [hunk 6]
```diff
@@ -155,16 +255,51 @@ async def replace_one(self, table: str, id: str, data: Any) -> Dict[str, Any]:
...
+        -------
+        :class:`Dict[str, Any]`
+            The new item.
+        """
         response = await self._request(
             method="PATCH",
             uri=f"/key/{table}/{id}",
             data=jsonlib.dumps(data),
         )
-
```
Comment (tsunyoku) [inline surrealdb/clients/http.py:163]: why are we removing these whitespaces? it separates out the logic and removing them is harder to read

File: surrealdb/clients/http.py [hunk 7]
```diff
@@ -110,43 +130,123 @@ async def _request(
...
+
+        Returns
+        -------
+        :class:`Dict[str, Any]`
+            The item selected.
+        """
         response = await self._request(method="GET", uri=f"/key/{table}/{id}")
         if not response.result:
             raise SurrealException(f"Key {id} not found in table {table}")
-
```
Comment (tsunyoku) [inline surrealdb/clients/http.py:142]: why are we removing these whitespaces? it separates out the logic and removing them is harder to read

File: surrealdb/clients/http.py [hunk 8]
```diff
@@ -110,43 +130,123 @@ async def _request(
...
+        -------
+        :class:`Dict[str, Any]`
+            The item created.
+        """
         response = await self._request(
             method="POST",
             uri=f"/key/{table}/{id}",
             data=jsonlib.dumps(data),
         )
-
```
Comment (tsunyoku) [inline surrealdb/clients/http.py:131]: why are we removing these whitespaces? it separates out the logic and removing them is harder to read

File: surrealdb/clients/http.py [hunk 9]
```diff
@@ -110,43 +130,123 @@ async def _request(
...
+        -------
+        :class:`Dict[str, Any]`
+            The items created
+        """
         response = await self._request(
             method="POST",
             uri=f"/key/{table}",
             data=jsonlib.dumps(data),
         )
-
```
Comment (tsunyoku) [inline surrealdb/clients/http.py:122]: why are we removing these whitespaces? it separates out the logic and removing them is harder to read

File: surrealdb/clients/http.py [hunk 10]
```diff
@@ -84,7 +102,9 @@ async def _request(
         uri: str,
         data: Optional[str] = None,
     ) -> SurrealResponse:
-        surreal_response = await self._http.request(method=method, url=uri, data=data)
+        surreal_response = await self._http.request(
+            method=method, url=uri, content=data
```
Comment (tsunyoku) [inline surrealdb/clients/http.py:106]: can you add a trailing comma on the end of this line so black multi-lines it?

**Review by tsunyoku — CHANGES_REQUESTED** [review]

File: surrealdb/clients/http.py (hunk 10 above)
Comment (CaedenPH) [reply surrealdb/clients/http.py:106]: This is what black formatted

File: surrealdb/clients/ws.py (hunk 1 above)
Comment (CaedenPH) [reply surrealdb/clients/ws.py:85]: In my opinion, it looks better. Lets have a discussion in the discord

File: surrealdb/clients/http.py (hunk 10 above)
Comment (tsunyoku) [reply surrealdb/clients/http.py:106]: yes, but adding a trailing comma will cause it to multi-line; pre-commit is set-up to automatically add trailing commas, but i suppose you are not using it

**Pull request closed without merging**